src/mmw/js/src/core/setup.js

```diff
@@ -7,6 +7,9 @@
 var $ = require('jquery');
 window.jQuery = window.$ = $;
 
+require('../../shim/backbone');
```

Comment:
Are you sure this doesn't need to be added here so that other parts of the application have access to them. Didn't we have trouble with this before.